src/Request/SearchByFeatureRequest.php

```diff
@@ -28,6 +30,7 @@ public function toJson(): string
                     'label' => $this->label,
                     'gender' => $this->gender,
                     'filters' => $this->filters,
+                    'maxNumResults' => $this->maxNumResults,
```

Comment:
will `null` not break things?

Reply:
null will not break anything because the array is the only argument to array_filter so the element will simply be removed